packages/runtime/src/errors/errors-list.ts

```diff
@@ -242,10 +242,16 @@ by an index that does not exist.`
     description: `lsig validation error`
   },
   INVALID_TRANSACTION_PARAMS: {
-    number: 27,
+    number: 37,
     message: "Secret key and Logic Signature should not be passed together",
     title: "Transaction params error",
     description: `Transaction params error`
+  },
+  UNKNOWN_TRANSACTION_TYPE: {
```

Comment:
should this be a TEAL error (as now we're using it within algob as well) ? We can move to a different class of errors later if you agree.

Reply:
yes, good idea. creating `transactionErrors` class